src/components/ActionList/components/Item/Item.tsx

```diff
@@ -120,7 +120,7 @@ export function Item({
   );
 
   return (
-    <li>
+    <li role={role === 'menuitem' ? 'presentation' : undefined}>
```

Comment:
This might just be me, but it took me a little investigation to wrap my brain around. I'm halfway wondering if a code comment might help to explain why it's there? Will wait for others to comment

Reply:
This isn't a necessary change, but was suggested by @svinkle.  Doing this makes sure that the `<li>` element is excluded for accessibility semantics.

Reply:
The expected, valid HTML structure:

```html
<ul role="menu">
  <li role="presentation">
    <button role="menuitem">Pay now</button>
  </li>
<ul>
```

Browser generated accessibility tree consumed by assistive technology:

```
- Menu
  - "Pay now", Menu item
```

The `li` is structural but is not required to convey the Menu pattern.